smarts/core/smarts.py

```diff
@@ -1281,7 +1283,18 @@ def fixed_timestep_sec(self) -> float:
     def fixed_timestep_sec(self, fixed_timestep_sec: float):
         if not fixed_timestep_sec:
             # This is the fastest we could possibly run given constraints from pybullet
-            self._rounder = rounder_for_dt(round(1 / MAX_PYBULLET_FREQ, 6))
+            self._rounder = rounder_for_dt(
+                round(
+                    1
+                    / config()(
+                        "physics",
+                        "max_pybullet_freq",
+                        default=MAX_PYBULLET_FREQ,
+                        cast=int,
+                    ),
```

Comment:
I think pulling this out as a separate variable would make the expression more readable.